src/components v2/Molecules/Forms/InputText/InputText.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <P2FormControl
+    v-bind="{
+      label,
+      inlineText,
+      isRequired,
+      extraMessage,
+      messageIcon,
+      error,
+      size,
+      disabled,
+      hideError,
+    }"
+  >
+    <P2InputText
+      :number="number"
+      :placeholder="placeholder"
+      :icon-left="iconLeft"
+      :icon-right="iconRight"
+      :centered="centered"
+      :error="error"
+      :disabled="disabled"
+      :value="value"
+      v-bind="$attrs"
```

Comment:
**issue:** we don't want a fallback $attrs binding.

Let's be explicit in our prop definitions. If we're missing something we have to define it in the props.